addons/project/static/src/js/project_kanban.js

```diff
@@ -92,12 +139,26 @@ const ProjectTaskKanbanRenderer = KanbanRenderer.extend({
         const grouped_by_date = ["date", "datetime"].includes(field.type);
         const grouped_by_m2m = field.type === "many2many";
         const readonly = !!field.readonly || !!fieldInfo.readonly;
+        const groupedByPersonalStage = (groupByFieldName === 'personal_stage_type_ids');
 
-        const draggable = !readonly && (!grouped_by_m2m || groupByFieldName == 'personal_stage_type_ids') &&
+        const draggable = !readonly && (!grouped_by_m2m || groupedByPersonalStage) &&
             (!grouped_by_date || fieldInfo.allowGroupRangeValue);
 
+        // When grouping by personal stage we allow any project user to create
+        let editable = this.columnOptions.editable;
+        let deletable = this.columnOptions.deletable;
+        if (['stage_id', 'personal_stage_type_ids'].includes(groupByFieldName)) {
+            this.groupedByM2O = groupedByPersonalStage ? true : this.groupedByM2O;
+            editable = groupedByPersonalStage ? true : this.isProjectManager;
+            deletable = groupedByPersonalStage ? true : this.isProjectManager;
+            this.createColumnEnabled = this.isProjectManager || groupedByPersonalStage;
```

Comment:
If I am not mistaken we could shorten to this, can't we?
```suggestion
            const allow_cud = this.isProjectManager || groupedByPersonalStage;
            this.createColumnEnabled = editable = deletable = allow_cud;
```